addins/CFSample/customfunctions.json

```diff
@@ -342,6 +342,35 @@
       "options": {
         "stream": true
       }
+    },
+    {
+      "id": "customErrorReturn",
+      "name": "customErrorReturn",
+      "description": "Custom errors as return value",
+      "helpUrl": "http://dev.office.com",
+      "parameters": [
+        {
+          "name": "errorCase",
+          "description": "error case",
+          "type": "number"
+        }
+      ]
+    },
+    {
+      "id": "customErrorReturnArray",
```

Comment:
Before you send out the next iteration, please do the following:
1. Test your manifest from a file share. (You'll need to make the JSON metadata available from some web site.) Make sure the new functions get registered, and you can execute them.
2. This add-in is 'non-shared runtime'. We try to keep parity between add-ins - same functionality in a shared runtime and in a non-shared runtime. Consult with the owners - @madhavagrawal17 and @ylu0826 which one is the corresponding add-in.

Reply:
got it, will ask @madhavagrawal17 and @ylu0826 for help

Reply:
@jiju-MS, Currently CFsample and CFsample2 (Shared Runtime) add-ins are not the same Custom functions so it is fine to not add this new API for CFsample2.

Reply:
Understand, so if we want to test Shared Runtime scenario, do we also need to update CFSample2?

Reply:
Yes. Perhaps not necessarily CFSample2, but one of the other CFSample add-ins that uses a Shared Runtime.